scripts/jenkins/jobs-ibs/openstack-mkcloud.yaml

```diff
@@ -0,0 +1,489 @@
+         description: the password for the user "openstack" as configured on the netapp server
+
+     - label:
+         name: label
+         default: openstack-mkcloud
+         description: |
+          Default: openstack-mkcloud | Possible values:
+          openstack-mkcloud-SLE11 (SLE11 host) | 
+          openstack-mkcloud-SLE12 (SLE12 host) | openstack-mkcloud-SLE12-aarch64 (SLE12 aarch64 host)
+
+     - string:
+         name: want_node_aliases
+         default: 
+         description: | 
+           List of aliases to assign to nodes. 
+           Takes all provided aliases and assign them to available nodes successively.
+           Examples want_node_aliases='controller=1,ceph=2,compute=1'
+           assigns the aliases to 4 nodes as controller, ceph1, ceph2, compute
+            want_node_aliases='data=1,services=2,storage=2'
+            assigns the aliases to 5 nodes as data, service1, service2,
+            storage1, storage2
+
+     - string:
+         name: scenario
+         default: 
+         description: | 
+          Defines which scenario file should be used. Only works with cloud5 or newer.
+          Currently only the 'batch' step uses such a file.
+
+     - string:
+         name: want_docker
+         default: 
+         description: set to 1 to deploy with docker
+
+     - string:
+         name: want_test_updates
+         default: 
+         description: allow disabling of test updates
+ 
+     - string:
+         name: want_magnum
+         default: 
+         description: Deploy with Magnum
+
+     - string:
+         name: want_barbican
+         default: 
+         description: Deploy with Barbican 
+
+     - string:
+         name: vcpus
+         default: '2'
+         description: number of vcpus for the compute nodes
+
+     - string:
+         name: want_sahara
+         default: 
+         description: set to 1 to deploy sahara
+
+     - string:
+         name: want_rootfs
+         default: 
+         description: set to btrfs to deploy with btrfs
+
+     - string:
+         name: want_postgresql
+         default: 
+         description: set to 1 to deploy with postgresql
+
+     - string:
+         name: want_efi
+         default: 
+         description: Set to UEFI boot admin node
+        
+     - string:
+         name: want_murano
+         default: 
+         description: set to 1 to deploy murano
+
+     - string:
+         name: want_horizon_integration_test
+         default:
+         description: Execute selenium integration tests for Horizon after Tempest.
+
+    wrappers:
+     - timestamps:
+     - build-name:
+         name: '#${BUILD_NUMBER}: ${ENV,var="cloudsource"} (${ENV,var="nodenumber"}/${ENV,var="networkingplugin"}/${ENV,var="tempestoptions"}/${ENV,var="mkcloudtarget"}/${ENV,var="github_pr"}${ENV,var="hacloud"})'
+     - timeout:
+        timeout: 3600
+        timeout-var: 'BUILD_TIMEOUT'
+        fail: true
+        type: absolute
+
+    builders:
+    - shell: |+
```

Comment:
Can you check if this script is OK after calling JJB? IIRC we need to scape certain things